express-crud-multer/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "express-crud-multer",
+  "version": "1.0.0",
+  "description": "CRUD-FileUpload using multer",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.20.2",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "multer": "^1.4.5-lts.1",
+    "nodemon": "^2.0.22"
```

Comment:
it should in dev dependency.

Reply:
move nodemon dependencies to dev Dependencies